src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -514,7 +513,7 @@ private async Task<bool> ShouldExitAfterDeterminingHostRuntime(bool isVerbose)
                     PrintVerboseForHostSelection(isVerbose, DotnetConstants.InProc6HostRuntime);
                 }
 
-                StartHostAsChildProcess(shouldNet8InProcBeLaunched);
```

Comment:
same with this; does this need to be removed?

Reply:
yeah that's my understanding that we need to just log and exit if an inproc payload is coming to OOP host.

Reply:
yes, will add this back